From 939c662ec8a221aedde213583e07e8c0cac810af Mon Sep 17 00:00:00 2001
From: Shiji Yang <yangshiji66@outlook.com>
Date: Thu, 19 Feb 2026 14:23:56 +0800
Subject: [PATCH] phy: mediatek: allow building T-PHY driver for MT7621 MIPS
 SoC

MT7621 also has compatible T-PHY which attached to the XHCI interface.

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
---
 drivers/phy/mediatek/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/phy/mediatek/Kconfig
+++ b/drivers/phy/mediatek/Kconfig
@@ -26,7 +26,7 @@ config PHY_MTK_XFI_TPHY
 
 config PHY_MTK_TPHY
 	tristate "MediaTek T-PHY Driver"
-	depends on ARCH_MEDIATEK || COMPILE_TEST
+	depends on ARCH_MEDIATEK || SOC_MT7621 || COMPILE_TEST
 	depends on OF && OF_ADDRESS
 	depends on HAS_IOMEM
 	select GENERIC_PHY
